src/lib/services/proposal/index.ts

```diff
@@ -55,76 +43,24 @@ import {
   getRelatedProposalsByContractAddress,
 } from "./api";
 import {
-  getDepositParamsLcd,
   getProposalDataLcd,
+  getProposalParamsLcd,
   getProposalsLcd,
-  getVotingParamsLcd,
 } from "./lcd";
 
-export const useGovParams = (): UseQueryResult<GovParams> => {
-  const lcdEndpoint = useLcdEndpoint();
-  const { data: assetInfos } = useAssetInfos({ withPrices: false });
-  const { data: movePoolInfos } = useMovePoolInfos({ withPrices: false });
-
-  const queryFn = useCallback(
-    () =>
-      Promise.all([
-        getDepositParamsLcd(lcdEndpoint),
-        getUploadAccessParamsLcd(lcdEndpoint),
-        getVotingParamsLcd(lcdEndpoint),
-      ]).then<GovParams>((params) => {
-        const minDepositParam = params[0].minDeposit[0];
-        const minDepositToken = coinToTokenWithValue(
-          minDepositParam.denom,
-          minDepositParam.amount,
-          assetInfos,
-          movePoolInfos
-        );
-        const minDepositAmount = deexponentify(
-          minDepositToken.amount,
-          minDepositToken.precision
-        ).toFixed() as Token;
+export const useProposalParams = () => {
+  const tier = useTierConfig();
+  const endpointApi = useBaseApiRoute("proposals");
+  const endpointLcd = useLcdEndpoint();
```

Comment:
Shall we use `apiEndpoint` and `lcdEndpoint` to make these accordingly with other files